.ocr/skills/references/final-template.md

```
@@ -0,0 +1,362 @@
+Full reviews available in session directory:
+
+| Reviewer | Blockers | Should Fix | Suggestions | File |
+|----------|----------|------------|-------------|------|
+| @principal-1 | 0 | 1 | 3 | `rounds/round-{n}/reviews/principal-1.md` |
+| @principal-2 | 0 | 0 | 2 | `rounds/round-{n}/reviews/principal-2.md` |
+| @quality-1 | 0 | 1 | 4 | `rounds/round-{n}/reviews/quality-1.md` |
+| @security-1 | 1 | 0 | 2 | `rounds/round-{n}/reviews/security-1.md` |
+
+**Session**: `.ocr/sessions/{session-id}/`
+**Discourse**: `rounds/round-{n}/discourse.md`
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)